packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -192,6 +193,8 @@ class AtClientImpl implements AtClient, AtSignChangeListener {
       _atChops ??= await _createAtChops(_atSign);
     }
 
+    // Set enrollment id
+    this.enrollmentId ??= enrollmentId;
```

Comment:
have this line in AtClientImpl._ instead of _init(). you can remove the param to _init